### PR TITLE
Add voice status feature

### PR DESCRIPTION
**Pull request by MinnDevelopment**
[contributing]: https://jda.wiki/contributing/contributing/

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [x] Internal code
- [x] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

Reference: https://github.com/discord/discord-api-docs/pull/6398

Allows setting a status for a voice channel.

### CODE REVIEW

**Comment by JustRed23** [comment]
Any update on when this will be available? Seems like a pretty nice feature!

**Comment by Xirado** [comment]
> Any update on when this will be available? Seems like a pretty nice feature!

"status: awaiting discord docs pr"

**Comment by JustRed23** [comment]
> > Any update on when this will be available? Seems like a pretty nice feature!
> 
> "status: awaiting discord docs pr"

Yea, but it has been for a while now..

**Comment by cybooo** [comment]
> > > Any update on when this will be available? Seems like a pretty nice feature!
> > 
> > 
> > "status: awaiting discord docs pr"
> 
> Yea, but it has been for a while now..

well ask discord to merge the docs pr.

File: src/main/java/net/dv8tion/jda/api/requests/Route.java [hunk 1]
```diff
@@ -203,6 +203,7 @@ public static class Channels
         public static final Route CREATE_PERM_OVERRIDE = new Route(PUT,    "channels/{channel_id}/permissions/{permoverride_id}");
         public static final Route MODIFY_PERM_OVERRIDE = new Route(PUT,    "channels/{channel_id}/permissions/{permoverride_id}");
         public static final Route DELETE_PERM_OVERRIDE = new Route(DELETE, "channels/{channel_id}/permissions/{permoverride_id}");
+        public static final Route SET_STATUS =           new Route(PUT, "channels/{channel_id}/voice-status");
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/api/requests/Route.java:206]:
```suggestion
        public static final Route SET_STATUS =           new Route(PUT,    "channels/{channel_id}/voice-status");
```

**Pull request merged**